### DISCUSSION TITLE
Unable to create a new remote session

### DISCUSSION THREAD

**Discussion by VelmuruganThangarasu** [appium/appium#18887] [Q&A] [answered]
I'm facing the error
 "org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource (WARNING: The server did not provide any stacktrace information)"

Please tell me, What causes this error? and what will be the solution?
![Unable to Create New Remote session](https://github.com/appium/appium/assets/53559866/5bcbd3bd-12ef-4981-bb5e-461badcdb481)

**Comment by KazuCocoa** [comment, marked as answer]
You should not add `/wd/hub` in the WebDriver URL by the client.
https://appium.io/docs/en/2.0/guides/migrating-1-to-2/#default-server-base-path may help to see some format differences between the old protocol and appium 2.0

**Reply by VelmuruganThangarasu** [reply]
Thank You! Now the error is resolved.!
